src/index.js

```diff
@@ -0,0 +1,29 @@
+import express from 'express';
+import dotenv from 'dotenv';
+import cors from 'cors';
+import productRoutes from './routes/productRoutes.js';
+import articleRoutes from './routes/articleRoutes.js';
+import commentRoutes from './routes/commentRoutes.js';
+import { error } from './middlewares/error.js';
+
+dotenv.config();
+
+const app = express();
+const PORT = process.env.PORT || 4000;
```

Comment:
필수는 아니지만, Node 계열로 동작하는 프레임워크의 경우 주로 3000번대의 port를 사용합니다.

```suggestion
const PORT = process.env.PORT || 3000;
```